tests/integration/rules/duo/duo_anonymous_ip_failure.json

```diff
@@ -1,68 +1,50 @@
-{
-  "records": [
-    {
-      "data": {
-        "username": "user.name@email.com",
-        "access_device": {
-          "flash_version": "27.0.0.0",
-          "java_version": "uninstalled",
-          "os_version": "10.12.6",
-          "browser_version": "60.0.0000.80",
-          "trusted_endpoint_status": "not trusted",
-          "os": "Mac OS X",
-          "browser": "Chrome"
-        },
-        "timestamp": 1505316499,
-        "new_enrollment": false,
-        "ip": "12.123.123.12",
-        "integration": "Test Integration",
-        "reason": "Anonymous IP",
-        "location": {
-          "city": "Place",
-          "state": "State",
-          "country": "US"
-        },
-        "factor": "Duo Push",
-        "device": "555-123-4567",
-        "result": "FAILURE"
+[
+  {
+    "data": {
+      "username": "user.name@email.com",
+      "access_device": {
+        "flash_version": "27.0.0.0",
+        "java_version": "uninstalled",
+        "os_version": "10.12.6",
+        "browser_version": "60.0.0000.80",
+        "trusted_endpoint_status": "not trusted",
+        "os": "Mac OS X",
+        "browser": "Chrome"
       },
-      "description": "Duo authentication log marked as failure as a result of 'Anonymous IP' that will create an alert",
-      "log": "duo:authentication",
-      "service": "stream_alert_app",
-      "source": "prefix_cluster_duo_auth_sm-app-name_app",
-      "trigger_rules": ["duo_anonymous_ip_failure"]
+      "timestamp": 1505316499,
```

Comment:
to confirm, you really only needed to have 2 entries here, `result` and `reason`, since that's only what you care to override, correct?: https://github.com/airbnb/streamalert/blob/master/rules/community/duo_authentication/duo_anonymous_ip_failure.py

Reply:
yep, I just left one of the remainder fields in